test/cases/api.sh

```diff
@@ -582,9 +612,16 @@ function waitForImgState() {
 
 #
 # Make sure that requesting a non existing paquet results in failure
+# or if TEST_MODULE_HOTFIXES flag is passed, we verify, that without that flag the resolve fails
 #
+
+if [ "$TEST_MODULE_HOTFIXES" != 1 ]; then
+  jq_edit = '.customizations.packages = [ "jesuisunpaquetquinexistepas" ]'
+else
+  jq_edit = 'del(.customizations.payload_repositories[] | select(.baseurl | match("*public/el8/el8-*-nginx-*")) | .module_hotfixes)'
+fi
 REQUEST_FILE2="${WORKDIR}/request2.json"
-jq '.customizations.packages = [ "jesuisunpaquetquinexistepas" ]' "$REQUEST_FILE" > "$REQUEST_FILE2"
+jq $jq_edit "$REQUEST_FILE" > "$REQUEST_FILE2"
```

Comment:
I don't have a strong preference here, but you could have just add the `module_hotfixes` as an additional test case, instead of conditionally testing that or the failure when selecting non-existing package. Running both should be reasonably fast, since the compose will fail on depsolving and no image will be built.